styles.css

```diff
@@ -159,3 +160,407 @@ header {
+  justify-content: space-between;
+  gap: 12px;
+  width: 327px;
+  height: 386px;
+  margin: 0;
+  background:
+    linear-gradient(
+      180.45deg,
+      rgba(38, 38, 38, 0) 0.75%,
+      rgba(38, 38, 38, 0.9) 61.94%
+    ),
+    url(./PImages/digitalmarket.png);
+  background-size: cover;
+}
+
+.cardworks3 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 12px;
+  width: 327px;
+  height: 386px;
+  margin: 0;
+  background:
+    linear-gradient(
+      180.45deg,
+      rgba(38, 38, 38, 0) 0.75%,
+      rgba(38, 38, 38, 0.9) 95.94%
+    ),
+    url(./PImages/postone1.png);
+  background-size: cover;
+}
+
+.cardworks4 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 12px;
+  width: 327px;
+  height: 386px;
+  margin: 0;
+  background:
+    linear-gradient(
+      180.45deg,
+      rgba(38, 38, 38, 0) 0.75%,
+      rgba(38, 38, 38, 0.9) 95.94%
+    ),
+    url(./PImages/buildingdesignconstruction.png);
+  background-size: cover;
+}
+
+.cardworks5 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 12px;
+  width: 327px;
+  height: 386px;
+  margin: 0;
+  background:
+    linear-gradient(
+      180.45deg,
+      rgba(38, 38, 38, 0) 0.75%,
+      rgba(38, 38, 38, 0.9) 95.94%
+    ),
+    url(./PImages/card-five.png);
+  background-size: cover;
+}
+
+.cardworks {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 12px;
+  width: 327px;
+  height: 386px;
+  margin: 0;
+  background: url(./PImages/Img\ Placeholderforprof.png);
+  background-size: cover;
+}
+
+.card1 {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 32px;
+  font-weight: 200;
+  line-height: 44px;
+  letter-spacing: 0.02em;
+  margin-top: 72px;
+  margin-left: 13px;
+  margin-right: 13px;
+  text-align: left;
+  color: #fff;
+}
+
+.cardtext {
+  font-family: 'Inter', serif;
+  font-size: 15px;
+  font-weight: 200;
+  line-height: 24px;
+  letter-spacing: 0.02em;
+  margin-left: 13px;
+  margin-right: 13px;
+  text-align: left;
+  color: #fff;
+  z-index: 222;
+}
+
+.card-button {
+  width: 247px;
+  height: 40px;
+  margin-left: 13px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  list-style: none;
+}
+
+.btn {
+  padding: 10px 12px;
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+}
+
+.projects {
+  width: 327px;
+  height: 48px;
+  left: 0;
+  top: 0;
+  border: none;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+/* About me Section */
+
+.about-section {
+  position: relative;
+  width: 375px;
+  background: #1c1a19;
+  padding-top: 120px;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-bottom: 50px;
+}
+
+.about-section-content {
+  font-family: "Inter", sans-serif;
+  color: #fff;
+  list-style: none;
```

Comment:
- [OPTIONAL] Good job adding a style none to these `li` items. You can do it too with your list of social icons in your main page to make it's look better